python/lsst/ts/ess/common/device/mock_device.py

```diff
@@ -33,6 +33,10 @@
 
 
 class MockDevice(BaseDevice):
+    # The wait time between sending telemetry (second). This can be adjusted by
+    # unit tests to mock connection timeouts.
+    wait_time = 1
```

Comment:
I gently suggest something like "telemetry_interval" would be clearer.

Reply:
Done